optimum/exporters/openvino/model_configs.py

```diff
@@ -3854,20 +3845,9 @@ def flatten_past_key_values(self, flattened_output, name, idx, t):
 class WhisperOpenVINOConfig(WhisperOnnxConfig):
     def patch_model_for_export(
         self, model: Union["PreTrainedModel", "TFPreTrainedModel"], model_kwargs: Optional[Dict[str, Any]] = None
-    ) -> ModelPatcher:
+    ) -> OVModelPatcher:
         return OVSeq2SeqModelPatcher(self, model, model_kwargs=model_kwargs)
 
-    @property
-    def inputs(self):
```

Comment:
why can this be removed ?

Reply:
because we are doing it correctly in optimum-onnx now ; seq2seq models used to make the decoder_input_ids dynamic only on the batch dim, supposing that the seq len dim will always be 1 when decoding, but it can be more (e.g. multi-turn chat).